assets/js/app.js

```diff
@@ -0,0 +1,59 @@
+/*
+ * App.js the main app for the base template.
```

Comment:
This should have been a rename of common,js or layout.js, I forgot which one.

Reply:
I think we don't use common.js anymore